src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -193,6 +193,11 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, ControlValue
    */
   @Input() alwaysRestoreSelectedOptionsMulti = false;
 
+  /**
+   * Recreate array of selected values
```

Comment:
```suggestion
   * Recreate array of selected values for multi-selects.
   *
   * This is useful if the selected values are stored in an immutable data structure.
```

please also add this to https://github.com/bithost-gmbh/ngx-mat-select-search/blob/master/README.md#inputs